docs/rules/padding-between-test-functions.md

```diff
@@ -0,0 +1,72 @@
+# Padding Between Test Functions (padding-between-test-functions)
```

Comment:
So what do you think about the rule name? As I understand it the rule enforces the padding around each test function and not specifically between two test functions. So maybe a better name would be `padding-around-test-functions`?